portfolio/src/main/java/com/google/sps/storage/CommentStorage.java

```diff
@@ -0,0 +1,15 @@
+package com.google.sps.storage;
+
+import java.util.ArrayList;
+import java.util.List;
+import com.google.sps.data.Comment;
+
+
+public interface CommentStorage {
+
+  void insert(Comment comment);
+
+  ArrayList<Comment> getNComments(int numComments, boolean sortAsc);
```

Comment:
two nits:
1. Having the interface declare that it returns a specific implementation is a little strange because you limit how implementing classes can be implemented. Do CommentStorage users really care what type of List is returned? or just that it's a List?
1. getN_ is a strange method name, how about listComments?